src/Games/NexusMods.Games.MountAndBlade2Bannerlord/Diagnostics/Diagnostics.cs

```diff
@@ -147,7 +147,7 @@ 3. The mod was installed from an outdated source
     internal static IDiagnosticTemplate ModVersionRangeTooHighTemplate = DiagnosticTemplateBuilder
         .Start()
         .WithId(new DiagnosticId(Source, 3))
-        .WithTitle("'{ModName}' Has a Too Recent Dependency with ID '{DependencyId}'")
+        .WithTitle("Mod Version Range Too High")
```

Comment:
Another strange title, this can be `Dependency too new` or `Unsupported Dependency Version`.